cmd/bot/main.go

```diff
@@ -149,7 +167,102 @@ func main() {
 	}
 }
 
+func isMessage(update map[string]any) bool {
+	_, hasMessage := update["message"]
+	return hasMessage
+}
```

Comment:
All of the Telegram-related functions (not just this one) should be moved to their own package.